front/lib/models/assistant/agent.ts

```diff
@@ -122,6 +123,10 @@ AgentConfiguration.init(
       type: DataTypes.INTEGER,
       allowNull: true,
     },
+    visualizationEnabled: {
+      type: DataTypes.BOOLEAN,
+      allowNull: false,
```

Comment:
Don't we need to set the default value here as well? Cause I see in the migration but not here.

Reply:
Not sure if we do, in the end it doesn't matter since it's in the migration (it will backfill), but explicit passing still required at typescript level. 
But will add it here as well to be safe 👍

Reply:
Yeah I'm not sure locally people run migrations, so in case they init through Sequelize, let's make sure it works.